eth_tester/backends/pyevm/main.py

```diff
@@ -507,6 +511,16 @@ def call(self, transaction, block_number="latest"):
             block_number,
         )
         if computation.is_error:
-            raise TransactionFailed(str(computation._error))
+            msg = str(computation._error)
+
+            # Check to see if it's a EIP838 standard error, with ABI signature
+            # of Error(string). If so - extract the message/reason.
+            if isinstance(computation._error, EVMRevert) and \
+               len(computation._error.args) > 0 and \
+               computation._error.args[0][:4] == b'\x08\xc3y\xa0':
```

Comment:
Can you move this string literal to a named constant at the top level of this module.